addons/project/migrations/12.0.1.1/post-migration.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, 'project', 'migrations/12.0.1.1/noupdate_changes.xml')
+    openupgrade.delete_records_safely_by_xml_id(
+        env, [
+            'project.msg_task_data_14_attach',
+            'project.msg_task_data_8_attach',
+            'project.msg_task_data_14',
+            'project.msg_task_data_8',
+            'project.project_task_data_8',
+            'project.mail_template_data_module_install_project',
```

Comment:
This is one is a fake one that you can remove. I think there will be a warning if not removed.